libraries/raylib55.c3l/raylib.c3i

```diff
@@ -777,7 +777,7 @@ fn void unloadVrStereoConfig(VrStereoConfig config) @extern("UnloadVrStereoConfi
 // NOTE: Shader functionality is not available on OpenGL 1.1
 fn Shader loadShader(ZString vsFileName, ZString fsFileName) @extern("LoadShader");          // Load shader from files and bind default locations
 fn Shader loadShaderFromMemory(ZString vs, ZString fs) @extern("LoadShaderFromMemory");      // Load shader from code strings and bind default locations
-fn bool isShaderValid(Shader shader) @extern("IsShaderReady") @extern("IsShaderValid");       // Check if a shader is ready
+fn bool isShaderValid(Shader shader) @extern("IsShaderReady");                               // Check if a shader is ready
```

Comment:
This should be 
```c
fn bool isShaderValid(Shader shader) @extern("IsShaderValid");  // Check if a shader is valid (loaded on GPU)
```

Reply:
Yes hehe... sorry just fixed quickly didnt think much ab it :p